### DISCUSSION TITLE
librewolf: command not found (AppImage)

### DISCUSSION THREAD

**Discussion by NoRainfallDuckDog** [netblue30/firejail#4499] [Q&A] [answered]

**Comment by kmk3** [comment, marked as answer]
@nntn-nntn [on Sep 2](https://github.com/netblue30/firejail/discussions/4499#discussion-3551276):

> firejail librewolf won't launch

Hello, this sounds like a bug, which should generally be reported as an issue
rather than in discussions.  Also, more information is needed.

Please go to the link below and follow the issue template:

<https://github.com/netblue30/firejail/issues/new?assignees=&labels=&template=bug_report.md&title=>

**Reply by kmk3** [reply]
Hmm now I see that you edited #4500 (GitHub doesn't notify for edits).  I'll
mark this as a duplicate of that one and reply in there then.
